processors/user-group.ftd

```diff
@@ -0,0 +1,31 @@
+-- ft.page: `user-groups` processor
+
+`user-groups` processor can be used to find all the [groups](/dev/user-group/)
+the current user belongs to.
+
+-- ft.code:
+lang: ftd
+
+\-- fpm.user-group-compat list groups:
+$processor$: user-groups
+
+-- ft.markdown:
+
+This will initialise the list `groups` with the groups user belongs to.
+
+-- ft.code: `fpm.user-group-compat` type
```

Comment:
I think definition of all such records should be in another file. We may re-use this definition in multiple processors. Soon we will have group-by-name processor for example which will also use this record. We shouldn't duplicate definition of the record in both documents but should have a link to it.